core/includes/common.inc

```diff
@@ -3048,6 +3048,19 @@ function backdrop_deliver_html_page($page_callback_result) {
     backdrop_add_http_header('X-Frame-Options', $frame_options);
   }
 
+  if ($site_config->get('block_interest_cohort')) {
+    $permissions_policy = backdrop_get_http_header('Permissions-Policy');
+    if (is_null($permissions_policy)) {
+      backdrop_add_http_header('Permissions-Policy', 'interest-cohort=()');
+    }
+    else {
+      // Only add interest-cohort if the header does not contain it already.
+      if (strpos($permissions_policy, 'interest-cohort') === FALSE) {
+        backdrop_add_http_header('Permissions-Policy', 'interest-cohort=()', TRUE);
```

Comment:
Doesn't that mean, that if the header already exists it will stay untouched - whatever value it has? Even a non-empty one? Shouldn't we replace it instead?

Reply:
This matches the D7 commit. And from what I can tell in the tests, it's just supposed to append this header not replace ones already there.

Reply:
> This matches the D7 commit

That means nothing at all - at least to me. :laughing: 

> from what I can tell in the tests, it's just supposed to append this header not replace

If that's the intention, then it works as designed. :+1: I wasn't sure.

Anyway, you can close this comment as resolved - my question is answered.